pylossless/pipeline.py

```diff
@@ -900,17 +1027,20 @@ def flag_ch_low_r(self):
         """
         # Calculate nearest neighbour correlation on
         # non-'manual' flagged channels and epochs...
+        logger.info('🏃🏃🏃 Starting flag_ch_low_r !')
         data_r_ch = self.get_n_nbr()[0]
 
         # Create the window criteria vector for flagging low_r chan_info...
-        flag_r_ch_inds = _detect_outliers(data_r_ch, flag_dim='ch',
-                                          init_dir='neg',
-                                          **self.config['ch_low_r'])
-
+        bad_ch_names = _detect_outliers(data_r_ch, flag_dim='ch',
+                                        init_dir='neg',
+                                        **self.config['ch_low_r'])
+        print('🧑‍🔧 flag_ch_low_r report!!!', bad_ch_names)
         # Edit the channel flag info structure
-        bad_ch_names = data_r_ch.ch[flag_r_ch_inds].values.tolist()
-        self.flagged_chs.add_flag_cat(kind='low_r', bad_ch_names=bad_ch_names)
-
+        self.flags["ch"].add_flag_cat(kind='low_r', bad_ch_names=bad_ch_names)
+        # TODO: CONFIRM THAT WE SHOULD BE RE-REFFING HERE (Scott)
```

Comment:
It seems like we are being inconsistent in our codebase about where we initiate a re-reference routine. Sometimes we put the call to `rereference` inside the flagged method. Sometimes we do a call to `rereference` in the `run` method just after another flagging method.
 
Just asset some consistency by always calling the re-reference method at the end of a flagging function.